packages/admin/src/components/Navigation/Navigation.js

```diff
@@ -96,7 +97,22 @@ const Navigation = ({modulesMenuTree, settingsMenuTree, menuOpen, onClick, activ
 
   const msg = id => intl.formatMessage({id})
 
-  return <StyledNav>
+  const handleCursorNavigation = key => {
+    const focusableElements = navigationEl.current.querySelectorAll('a, input')
+    const currentIndex = [...focusableElements].findIndex(el => document.activeElement.isEqualNode(el))
```

Comment:
what if `focusableElements` is empty or if the currently active element is not found in the `focusableElements` array for whatever reason? simply let it fail on line 105?